modules/core/src/passes/layers-pass.js

```diff
@@ -18,12 +18,12 @@ export default class LayersPass extends Pass {
     parameters = {},
     pass = 'draw',
     redrawReason = '',
-    customRender,
+    clearCanvas = true,
     effects,
     effectProps
   }) {
     const gl = this.gl;
-    if (!customRender) {
+    if (clearCanvas) {
```

Comment:
Not for this PR - FWIW, the luma.gl multi-pass system has a separate clear pass. This way it could be controlled by pass selection rather than adding props to the passes.